lib/screens/contributors.dart

```diff
@@ -22,7 +22,7 @@ class _ContributorsState extends State<Contributors>
   @override
   void initState() {
     super.initState();
-    getContributors(username: 'IEEE-VIT', repository: 'hacktoberfest-flutter')
+    getContributors(repository: widget.repoName)

```

Comment:
@Lomesh21 The primary aim of the devs of the repo was to retain the contributors of this project in contributors page,
so what we can do is shift that aim to the About Us page, with one click people can see the contributors of this repo and also visit, ik this is out of scope of this issue, but if u want to work on this I can create a separate issue or extend scope of this one le me knw

Reply:
Can you create new issue with more descriptive, I am happy to work on this.

Reply:
@Lomesh21 ok lets merge this one first and then I will create an issue for this